src/Network/Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -154,6 +154,101 @@ public CNM.PSVirtualRouter Convert(MNM.VirtualRouter source, CNM.PSVirtualRouter
             }
         }
 
+        private static void MapRouteTableV2sToRouteTables<MnmType, CnmType>(MnmType mnmObj, CnmType cnmObj)
+        {
+            /*
+             * MNM type Virtual Hub contains the property VirtualHubRouteTableV2s which
```

Comment:
did we figure out if we can mark existing commands as being deprecated eventually

Reply:
Looping in Anton @number213  , is it possible that we can mark these two cmdlets as being deprecated -> New-AzVirtualHubRoute and New-AzVirtualHubRouteTable ?

Reply:
Yes, there is
https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletdeprecationattribute

Reply:
Thank you , adding this to the mentioned cmdlets